fossa.c

```diff
@@ -698,6 +698,7 @@ static void ns_read_from_socket(struct ns_connection *conn) {
       }
     }
 #endif
+    (void)ret;			/* quiet compiler warning */
```

Comment:
Thanks.

BTW, `fossa.c` is a generated file. The change should be applied to `src/net.c`